test/testLauncher/composeConfig.test.js

```diff
@@ -97,12 +95,41 @@ describe('testLauncher readUserConfig', () => {
 			readRcConfig(),
 			{
 				test: 'test',
-				configs: [mockPath],
 				config: mockPath,
 			},
 		);
 	});
 
+	describe('Should work on linux platforms', function() {
```

Comment:
Also need cover cases for windows platfrom, becuase tests can be executed on different OS. Please add new tests for windows.